lib/postprocessing/image.js

```diff
@@ -231,6 +231,12 @@ function hasJpegQuality(metadata, instructions) {
     return true;
 }
 
+function needsOrientationApplied(metadata) {
+    if (typeof(metadata.Orientation) === "number" && metadata.Orientation > 1) {
```

Comment:
A valid orientation will always be between 1 and 8. So I would test if the value is a Number and `>1 && <9`.

Reply:
Done!